pkg/controller/operands/kubevirt.go

```diff
@@ -357,10 +357,26 @@ func toKvMediatedDevicesConfiguration(mdevsConfig *hcov1beta1.MediatedDevicesCon
 	}
 
 	return &kubevirtcorev1.MediatedDevicesConfiguration{
-		MediatedDevicesTypes: mdevsConfig.MediatedDevicesTypes,
+		MediatedDevicesTypes:    mdevsConfig.MediatedDevicesTypes,
+		NodeMediatedDeviceTypes: toKvNodeMediatedDevicesConfiguration(mdevsConfig.NodeMediatedDeviceTypes),
 	}
 }
 
+func toKvNodeMediatedDevicesConfiguration(hcoNodeMdevTypesConf []hcov1beta1.NodeMediatedDeviceTypesConfig) []kubevirtcorev1.NodeMediatedDeviceTypesConfig {
+	if len(hcoNodeMdevTypesConf) > 0 {
+		nodeMdevTypesConf := make([]kubevirtcorev1.NodeMediatedDeviceTypesConfig, 0, len(hcoNodeMdevTypesConf))
```

Comment:
Since we know the final length of the slice, we can maybe make it woth the known size and set the converted values with using the original index instead of using append?

Reply:
> Thanks for the PR. Can you also run deepcopy and openapi gen? If not, I can help with it.

I'm happy to do it, I just don't know how... maybe you can point me to what should I run?

Reply:
oh, it's only 2 elements.. what's wrong with append? it's pretty valid in go.

Reply:
This is a legit golang code. But I think that append is mostly for dynamic size slices, and in cases like this, I prefer direct assignment by the slice index. But that me and I won't block the PR for that (-: